contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryBase2_3.sol

```diff
@@ -100,7 +100,7 @@ abstract contract AutomationRegistryBase2_3 is ConfirmedOwner {
   uint256 internal s_fallbackGasPrice;
   uint256 internal s_fallbackLinkPrice;
   uint256 internal s_fallbackNativePrice;
-  uint256 internal s_expectedLinkBalance; // Used in case of erroneous LINK transfers to contract
+  uint256 internal s_reserveLinkBalance; // Unspent user deposits + unwithdrawn NOP payments
```

Comment:
That makes sense. Can we do this in a separate ticket/PR. This renaming touches a lot of existing logic, so I was thinking of having a separate PR for it, but feel it is okay to have it in this PR since it is just renaming. Given that we are actually changing the strucutre, I would propose to have it in its own PR.

Created https://smartcontract-it.atlassian.net/browse/AUTO-9250 and assigned to myself.